asyncjs/async-website
# Meetup - add to head, foot and latest talk

* Add Meetup link to latest talk
* Add Meetup link (with icon) to head
* Add Meetup link to foot

_With regard to adding the Meetup link to the latest talk, we discussed using Meetup to help increase awareness of Async rather than for controlling or monitoring numbers. I thought that the message should be welcoming rather than suggesting that people are required to sign up via Meetup, as this might potentially put some people off._

<img width="832" alt="Screenshot 2019-07-10 10 35 29" src="https://user-images.githubusercontent.com/341682/60958767-d1fe2f00-a2fe-11e9-8693-01b8e3bbb7bd.png">

<img width="835" alt="Screenshot 2019-07-10 10 35 29(2)" src="https://user-images.githubusercontent.com/341682/60958774-d7f41000-a2fe-11e9-819d-2498f4152f2d.png">
